### PR TITLE
Tensorflow nn frontend avg_pool1d

### PR DESCRIPTION
**Pull request by ajithpious**
Pull request for #11775

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/nn.py [hunk 1]
```diff
@@ -482,3 +482,11 @@ def relu6(features, name=None):
 @to_ivy_arrays_and_back
 def softmax(logits, axis=None, name=None):
     return ivy.softmax(logits, axis=axis)
+
```
Comment (ZoeCD) [inline ivy/functional/frontends/tensorflow/nn.py:485]: Hi! A small lint error `E302 expected 2 blank lines, found 1` Could you please fix that really quick?

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py [hunk 2]
```diff
@@ -1270,3 +1270,29 @@ def test_tensorflow_relu6(
         on_device=on_device,
         features=x[0],
     )
+@handle_frontend_test(
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1273]: There are some lint errors here: 
```
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1256:1: F811 redefinition of unused 'test_tensorflow_relu6' from line 1165
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1273:1: E302 expected 2 blank lines, found 0
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1275:13: E251 unexpected spaces around keyword / parameter equals
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1275:15: E251 unexpected spaces around keyword / parameter equals
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1277:43: E[23](https://github.com/unifyai/ivy/actions/runs/4350476086/jobs/7601211193#step:5:24)1 missing whitespace after ','
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1282:89: E501 line too long (122 > 88 characters)
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1290:18: E[25](https://github.com/unifyai/ivy/actions/runs/4350476086/jobs/7601211193#step:5:26)1 unexpected spaces around keyword / parameter equals
ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py:1[29](https://github.com/unifyai/ivy/actions/runs/4350476086/jobs/7601211193#step:5:30)1:17: E251 unexpected spaces around keyword / parameter equals
```
Did you install the [pre-commit hook](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit)?

**Review by ZoeCD — CHANGES_REQUESTED** [review]
Additionally, the test is failing. Please review the details in the intelligent-tests section. Let me know if you have any questions! 😊

File: ivy/functional/frontends/tensorflow/nn.py (hunk 1 above)
Comment (ajithpious) [reply ivy/functional/frontends/tensorflow/nn.py:485]: sure

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py (hunk 2 above)
Comment (ajithpious) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py]: Yes i did run precommit

**Pull request closed without merging**